pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>software.amazon.sns</groupId>
     <artifactId>sns-extended-client</artifactId>
-    <version>1.0.0</version>
+    <version>2.0.0</version>
```

Comment:
I believe new version of payload-offloading-java-common-lib-for-aws is published. Can you please update the file to use the new version on that library?

```
<dependency>
            <groupId>software.amazon.payloadoffloading</groupId>
            <artifactId>payloadoffloading-common</artifactId>
            <version>1.0.0</version>.  ===> should be changed to <version>2.0.0</version>
</dependency>
```

Reply:
updated the payload-offloading-java-common-lib-for-aws version to v 2.1.2